app/src/main/java/com/asfoundation/wallet/manage_wallets/ManageWalletViewModel.kt

```diff
@@ -1,23 +1,26 @@
 package com.asfoundation.wallet.manage_wallets
 
-import android.text.format.DateUtils
 import androidx.lifecycle.ViewModel
 import com.asfoundation.wallet.home.usecases.DisplayChatUseCase
+import com.asfoundation.wallet.ui.wallets.WalletBalance
+import com.asfoundation.wallet.ui.wallets.WalletDetailsInteractor
+import com.asfoundation.wallet.ui.wallets.WalletsInteract
 import com.asfoundation.wallet.wallets.domain.WalletInfo
 import com.asfoundation.wallet.wallets.usecases.ObserveWalletInfoUseCase
 import dagger.hilt.android.lifecycle.HiltViewModel
 import io.reactivex.Observable
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
-import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
 @HiltViewModel
 class ManageWalletViewModel
 @Inject
 constructor(
   private val displayChatUseCase: DisplayChatUseCase,
-  private val observeWalletInfoUseCase: ObserveWalletInfoUseCase
+  private val observeWalletInfoUseCase: ObserveWalletInfoUseCase,
+  private val walletsInteract: WalletsInteract,
+  private val walletDetailsInteractor: WalletDetailsInteractor,
 ) : ViewModel() {
 
   private val _uiState = MutableStateFlow<UiState>(UiState.Idle)
```

Comment:
Can the _uiState and uiState be only one var?

Reply:
it's a pattern for that to avoid external changes on _uiState